news.xml

```diff
@@ -291,7 +291,7 @@
       We're happy to announce that <strong>NixCon 2017</strong>, the
       second Nix Conference, will take place <strong>October 28–31 2017 in Munich</strong>
       For more information, see the
-      <a href="http://nixcon2017.org/">NixCon 2017 website</a>.
```

Comment:
http://nixcon2017.org/ still works for me

Reply:
static site that won’t change